lib/id3c/api/static/index.html

```diff
@@ -91,7 +91,12 @@ <h3 class="code">GET /v1/warehouse/identifier-sets/<em>&lt;name&gt;</em></h3>
 
     <h3 class="code">PUT /v1/warehouse/identifier-sets/<em>&lt;set&gt;</em></h3>
     <p>Make a new identifier set <em>name</em>.  Idempotent if the set already
-    exists.  An optional <em>description</em> parameter may be provided.
+    exists.  Optional <em>description</em> parameter may be provided. A <em>use</em>
+    parameter is required for new sets and is optional if only updating
+    <em>description</em> on an existing set.
```

Comment:
I think it's still idempotent in the sense that it if you call it multiple times with the same arguments, it will not update the record on each call. If something changed (either a new record, or existing record where use or description changed) it updates the record and returns a 200 code, but if everything is the same as the existing record it returns a 204 (No Content)

Reply:
I think youre right. 👍